src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf

```diff
@@ -159,8 +159,8 @@ resource "helm_release" "jupyterhub" {
             login_service        = "Keycloak"
             username_claim       = "preferred_username"
             claim_groups_key     = "groups"
-            allowed_groups       = ["/analyst", "/developer", "/admin"]
-            admin_groups         = ["/admin"]
+            allowed_groups       = ["/analyst", "/developer", "/admin", "jupyterhub_admin", "jupyterhub_developer"]
+            admin_groups         = ["/admin", "jupyterhub_admin"]
```

Comment:
Hi @aktech wouldn't this create conda-store namespaces and EFS share files for those then when a users receives such group?